lib/modules/manager/devcontainer/readme.md

```diff
@@ -0,0 +1,6 @@
+Extracts all Docker images from the `image` and `features` properties in these files:
+
+- `.devcontainer.json`
+- `.devcontainer/devcontainer.json`
+
+The `devcontainer` manager does _not_ support `build.dockerFile` or `dockerComposeFile` values as these are covered by the [dockerfile](../dockerfile/readme.md) and [docker-compose](../docker-compose/readme.md) managers respectively.
```

Comment:
```suggestion
The `devcontainer` manager does _not_ support `build.dockerFile` or `dockerComposeFile` values as these are covered by the [`dockerfile`](../dockerfile/readme.md) and [`docker-compose`](../docker-compose/readme.md) managers respectively.
```

We use the `code` styling in the docs when referring to Renovate's managers. So we should match that.